src/Illuminate/Routing/ResponseFactory.php

```diff
@@ -69,6 +69,18 @@ public function noContent($status = 204, array $headers = [])
         return $this->make('', $status, $headers);
     }
 
+    /**
+     * Create a new "accepted" response.
+     *
+     * @param  mixed  $content
+     * @param  array  $headers
+     * @return \Illuminate\Http\Response
+     */
+    public function accepted(mixed $content = '', array $headers = [])
```

Comment:
Need to remove the `mixed` type hint here. The other methods don't do that. The `array` is good.
```
public function accepted($content = '', array $headers = [])
```